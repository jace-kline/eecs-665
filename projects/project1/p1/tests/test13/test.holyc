"Testing escaping a tab literal in char"
'\  
"Testing escaping a space literal in char"
'\ 
"Testing a tab literal in char"
'   
"Testing a space literal in char"
' 