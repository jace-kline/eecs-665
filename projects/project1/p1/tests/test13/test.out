STRINGLIT:"Testing escaping a tab literal in char" [1,1]
CHARLIT: 
STRINGLIT:"Testing escaping a space literal in char" [3,1]
CHARLIT: 
STRINGLIT:"Testing a tab literal in char" [5,1]
CHARLIT: 
STRINGLIT:"Testing a space literal in char" [7,1]
CHARLIT: 
EOF [8,3]
